Bottom: 535e298f0bcabe6f107bdb1f82393b046ff0a672
Top:    05a0eb32edc286b12f857deab71e245408f68cb0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-24 18:12:31 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index ece3723..680d74a 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -296,7 +296,6 @@ void ghid_get_pointer (gint *, gint *);
 
 /* gui-output-events.c function prototypes.
 */
-
 void ghid_port_ranges_changed (void);
 void ghid_port_ranges_zoom (gdouble zoom);
 gboolean ghid_port_ranges_pan (gdouble x, gdouble y, gboolean relative);
